app/experimenter/experiments/api/v4/serializers.py

```diff
@@ -53,24 +70,14 @@ class Meta:
         )
 
     def get_bucketConfig(self, obj):
-        return {
-            "randomizationUnit": "normandy_id",
-            "namespace": "",
-            "start": 0,
-            "count": 0,
-            "total": 10000,
-        }
```

Comment:
Also I got rid of this because we shouldn't really emit non sensical invalid bucket range definition before the real one is created.